spec/repository_spec.rb

```diff
@@ -26,3 +26,17 @@
     expect(EXTRA_FILES).to eq([])
   end
 end
+
+describe "The file \"renamed-styles.json\"" do
+  it "must be valid JSON" do
+    expect(JSON_IS_VALID).to be true
+  end
```

Comment:
> Style.load also parses the CSL in Ruby (that's not necessary for validation) so we'd save more

We'd save less, no? I just showed that `Style.load` is only ca. 7 seconds slower than no validation at all.